### ISSUE TITLE
Radio table columns settings: UI for custom columns is not accessible for screen readers

### ISSUE THREAD

**Issue by TomaszGasior** [TomaszGasior/RadioLista-v3#207]
- Disabled/enabled state of each radio table column is not announced by screen readers. 
- The whole widget does not use list (ul/ol tag) while it looks like list.

**Comment by NightPurrer** [comment]
I saw that you need some testers for fixing the accessibility issues. If you want, can we contact privately via audio call on any app? I can try help fix it by checking what's work or not work but talking and doing it step by step could be better and more facilitating to do it :).
